src/actions/forms.js

```diff
@@ -63,18 +63,18 @@ export function retrieveStreamMedia(audioFile, mediaPlayer) {
       // Bind media player
       hls.attachMedia(mediaPlayer.current);
       // MEDIA_ATTACHED event is fired by hls object once MediaSource is ready
-      hls.on(Hls.Events.MEDIA_ATTACHED, function() {
+      hls.on(Hls.Events.MEDIA_ATTACHED, function () {
         hls.loadSource(audioFile);
         // BUFFER_CREATED event is fired when fetching the media stream is successful
-        hls.on(Hls.Events.BUFFER_CREATED, function() {
+        hls.on(Hls.Events.BUFFER_CREATED, function () {
           dispatch(streamMediaSuccess());
         });
       });
 
       // ERROR event is fired when fetching media stream is not successful
-      hls.on(Hls.Events.ERROR, function(event, data) {
+      hls.on(Hls.Events.ERROR, function (event, data) {
         let errorCode = null;
-        if (data.fatal) {
+        if (data.fatal && data.type !== Hls.ErrorTypes.OTHER_ERROR) {
```

Comment:
Could you please add a comment here explaining the change?  It might be good to even reference the upstream issue for more information and to help determine when it can be removed in the future.